assets/styles/bootstrap/forms.less

```diff
@@ -510,7 +510,7 @@ input[type="checkbox"] {
   // `position: relative;` on them. Also accounts for the grid gutter padding.
   .has-feedback .form-control-feedback {
     top: 0;
-    right: (@grid-gutter-width / 2);
+    right: -20px; //original value: (@grid-gutter-width / 2);
```

Comment:
Could you put this outside of this file, because I'll end up overwriting it if I upgrade bootstrap.